js/src/redux/providers/signerMiddleware.js

```diff
@@ -120,7 +120,7 @@ export default class SignerMiddleware {
         });
     }
 
-    handlePromise(this._api.signer.confirmRequest(id, { gas, gasPrice }, password));
+    handlePromise(this._api.signer.confirmRequest(id, { gas, gasPrice }, password.password));
```

Comment:
Sure, but the issue is comes from components passing different data types to _callback props_.
Actually, it's `views/Signer/components/SignRequest/signRequest.js` and `views/Signer/components/TransactionPending/transactionPending.js` that have different `onConfirm` method _signature_/argument types